src/components/space-between.tsx

```diff
@@ -1,6 +1,7 @@
 import { Stack, StackProps } from '@stacks/ui';
 import React from 'react';
+import { forwardRefWithAs } from '@stacks/ui-core';
 
-export const SpaceBetween: React.FC<StackProps> = props => (
-  <Stack isInline alignItems="center" justifyContent="space-between" {...props} />
-);
+export const SpaceBetween = forwardRefWithAs<StackProps, 'div'>((props, ref) => (
+  <Stack isInline alignItems="center" justifyContent="space-between" {...props} ref={ref} />
+));
```

Comment:
Should this moved to the UI library?

Reply:
Yeah I was thinking so, I typically have 2-3 components like this that I implement in each and find useful:
`<SpaceBetween>` and `<Centered>` come to mind